batch_inference_v2/batch_inference_v2.py

```diff
@@ -167,6 +171,12 @@ def infer(
     context.logger.info(f"Loading model...")
     if isinstance(model_path, mlrun.DataItem):
         model_path = model_path.artifact_url
+    if not mlrun.datastore.is_store_uri(model_path):
+        raise mlrun.errors.MLRunInvalidArgumentError(
+            f"The provided model path is an invalid store uri: {model_path}, should start with store://. "
```

Comment:
```suggestion
            f"The provided model path ({model_path}) is an invalid store uri - it should start with `store://`. "
```

Reply:
If we use double brackets, the provided value won't be shown in the err message (you will get `{model_path}` instead of the actual model_path value such as `v3io://...`
In addition, if we add ` `store://`, it will present it as a clickable link in the err message which is incorrect, therefore I have decided to keep it in the original format.